mmcv/ops/points_in_polygons.py

```diff
@@ -0,0 +1,31 @@
+import torch
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext('_ext', ['points_in_polygons_forward'])
+
+
+def points_in_polygons(points, polygons):
+    """Judging whether points are inside polygons.
+
+    Args:
+        points (torch.Tensor): It has shape (B, 2), indicating (x, y).
+        polygons (torch.Tensor): It has shape (M, 8), indicating
+            (x1, y1, x2, y2, x3, y3, x4, y4).
+
+    Returns:
+        torch.Tensor: Return the result with the shape of (B, M),
+            1 indicates that the point is inside the polygon,
+            0 indicates that the point is outside the polygon.
```

Comment:
```suggestion
        torch.Tensor: Return the result with the shape of (B, M),
        1 indicates that the point is inside the polygon,
        0 indicates that the point is outside the polygon.
```

Reply:
Perhaps we can explain more specifically the meaning of the dimension M.

Reply:
OK